pkg/apiserver/crud/handler.go

```diff
@@ -55,3 +56,15 @@ func getApiViewFromHeader(reqHeaders http.Header) string {
 
 	return DefaultApiView
 }
+
+const mysqlDuplicateEntryError = 1062
+
+func IsDuplicateEntryError(err error) bool {
+	if err != nil {
+		mysqlErr, ok := err.(*mysql.MySQLError)
```

Comment:
did you see the error type in your IDE? i couldn't quickly find out which type it actually is, but wouldn't have expected it to be a pointer type